mantidproject/mantid
# Add IPython Console to Workbench

Description of work.

This adds an IPython console to workbench as a plugin and sets a default layout for the dock widgets.

**To test:**

* Build and start workbench. See that the IPython console is on the left and the log window on the right.
* Test that IPython works and that long runnning operations do not log the GUI.

Refs #21251

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
